api/src/main/java/org/openmrs/module/labintegration/api/hl7/openelis/OpenElisHL7Config.java

```diff
@@ -47,6 +47,11 @@ public OrderIdentifier buildOrderIdentifier(Order order) {
 		return new OpenElisOrderIdentifier(order);
 	}
 	
+	@Override
+	public String getSendingFacilityNamespaceID() {
+		return null;
```

Comment:
Consider using the UnsupportedOperationException or other.

Reply:
It's not UnsupportedOperation in this case. For OpenElis messages this field is just not needed, so returning null here results in empty string in data in the message.

Reply:
Okay.